.github/wiki/Project-Dependencies.md

```diff
@@ -0,0 +1,62 @@
+To simplify setup for developers new to the application, as well as having a reliable build system that is also able to run past builds again in a reproducible fashion. The creation and set up of a new project will be faster and safer requiring only the language runtime (`Ruby`) and dependency manager installed as prerequisites.
+
+ The project normally contains:
+
+- Fastlane: is the easiest way to automate beta deployments and releases for the `iOS` (also `Android`) applications. 🚀 It handles all tedious tasks, like generating screenshots, dealing with code signing, and releasing the application.
```

Comment:
```suggestion
- Fastlane: is the easiest way to automate beta deployments and releases for the `iOS` (also `Android`) applications. 🚀 It handles all tedious tasks, such as generating screenshots, dealing with code signing, and releasing the application.
```

Reply:
Fixed in https://github.com/nimblehq/ios-templates/pull/271/commits/9db8df6e06b41151e5d0e99e502d821e1bd8ea48